openshift-gitops/operators/gitops-operator.yaml

```diff
@@ -8,4 +8,8 @@ spec:
   installPlanApproval: Automatic
   name: openshift-gitops-operator 
   source: redhat-operators 
-  sourceNamespace: openshift-marketplace
\ No newline at end of file
+  sourceNamespace: openshift-marketplace
+  config:
+    env: 
+      - name: DISABLE_DEFAULT_ARGOCD_INSTANCE
+        value: "true"
```

Comment:
Thanks

Reply:
Now that you have the argocd.yaml along with the gitops-operator.yaml, the name openshift-gitops/operators will something like install and it should also move into the infra subfolder. The scripts/subcriptions.sh will also move inside openshift-gitops/infra

Reply:
I will move it under openshift-gitops/argocd as it manages the argocd subscription and deployment
The scripts folder is part of the suggested folder structure for the validated-patterns, maybe I can create a scripts/infra subfolder?